core/src/tpu.rs

```diff
@@ -148,6 +148,8 @@ impl Tpu {
 
         let (non_vote_sender, non_vote_receiver) = banking_tracer.create_channel_non_vote();
 
+        const MAX_STREAMS_PER_MS_TPU: u64 = DEFAULT_MAX_STREAMS_PER_MS;
```

Comment:
Any reason to define a new constant here, instead of using the default constant value?

Reply:
Not exactly -- this was just expecting we might have different PPS for tpu and tpu forward in the future.

Reply:
done